pkg/apis/app/v1alpha1/kogitoservices_types.go

```diff
@@ -188,6 +189,12 @@ type KogitoServiceSpec struct {
 	// Infra provides list of dependent KogitoInfra objects.
 	// +optional
 	Infra []string `json:"infra,omitempty"`
+
+	// Create Service monitor instance to connect with Monitoring service
+	// +operator-sdk:gen-csv:customresourcedefinitions.specDescriptors=true
+	// +operator-sdk:gen-csv:customresourcedefinitions.specDescriptors.displayName="Monitoring"
+	// +operator-sdk:gen-csv:customresourcedefinitions.specDescriptors.x-descriptors="urn:alm:descriptor:com.tectonic.ui:label"
```

Comment:
This won't work since it's a data structure and not a String. I need to take a look at this actually to see if something changed in the new OLM version.

Reply:
I had removed these extra comments. Just keep first line